### PR TITLE
Taylor - Signup Form validation & Binding Signup Function 

### PR DESCRIPTION
**Pull request by TaylorDJones11**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Other: Form Validation & Binding Signup Function
  
  
## Description of what this Pull Request does
- Adding form validation for the signup form
- Binding the Firebase signup function to the form component
- 

## Related Tickets & Documents
-Closes: #113 

  
## QA Instructions, Screenshots, Recording

- ROADBLOCKS
- I changed the Link to a button because I wasn't able to catch the form validation otherwise, it would send me straight to the main page. 
- I lost one aspect of the toggle icon function, when the password is visible in the input and a user erases it completely, the icon no longer goes back to the default state of the open icon. I displayed this in the last part of the gif.

SignupForm

### CODE REVIEW

File: src/composables/signupFormValidation.js
Comment (annonymousauthority) [inline src/composables/signupFormValidation.js:1]: Didn't @DeborahIhesiaba create a form validation function?

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Look at the validation function properly

File: src/composables/signupFormValidation.js
Comment (TaylorDJones11) [reply src/composables/signupFormValidation.js:1]: Yes, but it only covers email/password. Do I then add to her function to cover first name, last name, username, and confirm password?

Comment (annonymousauthority) [reply src/composables/signupFormValidation.js:1]: Yes please, let's have one area of concern

**Comment by TaylorDJones11** [comment]
> Look at the validation function properly

What exactly am I looking for? Am I not catching the errors correctly? Adding more edge cases like "user already exists"?

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**